File: cpp/class_student.cpp
#include<iostream>
using namespace std;
class Student
{
private :
string name, roll, mobile;
public:
	Student(string name1, string roll1, string mobile1)
	{
		name=name1;
		roll=roll1;
		mobile=mobile1;
	}
	void display()
	{
		cout<<"name"<<name <<"\n" <<"roll"<<roll << "\n" <<"mobile"<<mobile <<"\n";
	}
};
int main()
{


	Student s1=Student("xyz", "123", "1234567890");
	s1.display();

  	string name, roll, mobile;
	cout<<"Enter Student name ";
	cin>>name;
	cout<<"Enter Student roll ";
	cin>>roll;
	cout<<"Enter Student mobile number ";
	cin>>mobile;

	Student s2=Student(name, roll, mobile);
	s2.display();
	return 0;
}

File: python/programs/class_student.py

class Student:
	def __init__(self, name1,  roll1,  mobile1):
		self.name=name1
		self.roll=roll1
		self.mobile=mobile1

	def display(self):
		print(f"name {self.name} \n roll {self.roll} \n mobile {self.mobile} \n")
# main
s1=Student("xyz", "123", "1234567890")
s1.display()
name =input("Enter Student name ") 
roll= input("Enter Student roll ")
mobile=input("Enter Student mobile number ")

s2=Student(name, roll, mobile)	
s2.display()
	
 